hack/jenkins/test-cmd-dockerized.sh

```diff
@@ -20,10 +20,10 @@ set -o pipefail
 set -o xtrace
 
 # Runs test-cmd, intended to be run in prow.k8s.io
-set -x;
 
 # TODO: make test-cmd should handle this automatically
 source ./hack/install-etcd.sh
```

Comment:
before this change we wind up debugging every shell line from install-etcd.sh, which is excessive, it has output.

after this change we still log the make commands, but those are subprocesses that ignore -x